modules/virt-reclaiming-persistent-volumes-for-binding-to-a-new-persistent-volume.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// virt/virtual_machines/virtual_disks/virt-reclaiming-persistent-volumes.adoc
+
+[id="virt-reclaiming-persistent-volumes-for-binding-to-a-new-persistent-volume_{context}"]
+
+= Reclaiming persistent volumes for binding to a new persistent volume
+
+Use this procedure to:
+
+* Release the persistent volume from the persistent volume claim.
+* Erase all data in the share.
+* Create a new persistent volume with a different name,
```

Comment:
~a new~

s/name, maintaining/name that maintains

Reply:
Create a persistent volume that uses a different volume name. When creating the
new volume, use the same connection details as the deleted persistent volume.

This is in the actual procedure as the location above has been omitted per your other comment below.